kolibri/core/assets/src/views/ExamReport/index.vue

```diff
@@ -187,6 +212,16 @@
         }
         return null;
       },
+      currentAttempt() {
+        return this.attemptLogs.find(a => a.item === this.itemId);
+      },
+      currentAttemptDiff() {
+        return this.currentAttempt &&
+          this.currentAttempt.diff &&
+          this.currentAttempt.diff.correct !== null
+          ? this.currentAttempt.diff
+          : null;
+      },
     },
     methods: {
       handleNavigateToQuestion(questionNumber) {
```

Comment:
Getting a `TypeError` when this handler is invoked, because the EmbeddedMultiPaneLayout component does not have a `scrollMainToTop` public method - probably simplest to copy the implementation across for consistency?